lang/rust/rust.talon

```diff
@@ -61,7 +61,6 @@ state (mod | module): "mod "
 state ref (mute | mutable): "ref mut "
 state ref: "ref "
 state trait: "trait "
-state match: user.code_state_switch()
```

Comment:
We probably need to deprecate these voice commands and not just remove them.

Reply:
No. They already exist as part of the standard imperative grammar. Those were duplicates.

Reply:
The standard imperative command for switch statements is state (switch | match): user.insert_snippet_by_name("switchStatement"). 
Talon was not even using the language specific ones for me in the first place.